README.md

```diff
@@ -2,13 +2,64 @@
 
-## Quickstart
+- [The ItchySats Roadmap](https://itchysats.medium.com/itchysats-roadmap-to-the-most-awesome-bitcoin-dex-464a42bf4881) which includes a simplified example of how the protocol works
+- [Blogpost](https://comit.network/blog/2022/01/11/cfd-protocol-explained) by @luckysori of the COMIT team, describing the protocol in detail
+- [DLC overview](https://bitcoinops.org/en/topics/discreet-log-contracts/) on Bitcoin Optech for information beyond ItchySats
 
-All the components can be started at once by running the following script:
+## Quickstart: Users / Traders
+
+This guide is for how to use ItchySats as a taker.
+Currently, ItchySats does not officially support multiple makers yet.
+Please note that trade execution is already fully peer to peer, but there is currently only one maker out there.
+We are working on adding support for maker discovery and trading with multiple makers for a more competitive market.
+
+ItchySats is available in the [Umbrel](getumbrel.com/) appstore.
+We recommend using ItchySats on Umbrel at the moment.
+
+If you want to try ItchySats without Umbrel you can use the [latest binary](github.com/itchysats/itchysats/releases/latest) or [latest docker container](https://github.com/itchysats/itchysats/pkgs/container/itchysats%2Ftaker) but that might not be as straightforward.
+
+To open a position:
+
+1. Install the App on Umbrel (or start the binary / container)
+2. Transfer funds into the ItchySats wallet
+3. Open a `long` or `short` position - this will lock the funds in a multisig on chain
+4. Wait for the price to move
+5. Close the position - this will spend the locked up funds according to the price
+
+With the current maker you can close positions at any point in time.
+Trades are limited to a contract size between `100` and `1000` contracts.
+
+If the market maker is not available you can close by using an independent oracle attesting to the price.
+Oracle price outcomes can be found [here](https://outcome.observer/h00.ooo/x/BitMEX/BXBT).
+
+**Can't find what you are looking for? Check out the [👉 FAQ 👈](http://faq.itchysats.network).**
+
+### ItchySats Wallet
+
+ItchySats includes an internal wallet that is used to sign transactions during the DLC setup.
+Additionally, when a CFD is closed, your payout is sent to an address owned by this wallet.
+This wallet is completely under your control.
+You can withdraw from the wallet ay any time.
+
+On Umbrel this wallet is derived from the Umbrel Seed, so the only thing you have to back up is the Umbrel seed.
+
+When running the binary / docker container a random seed will be used to derive the wallet.
+Make sure to back up the `taker_seed` file that can be found in the data directory of the application.
+
+### Safety
+
+ItchySays is currently Beta software.
+We are doing our best to make ItchySats stable, but there could be unexpected bugs that result in losing funds or positions being closed at an unfavorable point in time.
```

Comment:
With loss of funds, do you mean locked up forever by bad scripts or something? I think this should be reworded to only focus on the latter part (closed positions at unfavorable points in time).